sdk/go/auto/local_workspace_test.go

```diff
@@ -66,7 +66,7 @@ type mockPulumiCommand struct {
 	stderr       string
 	exitCode     int
 	err          error
-	capturedArgs []string
+	capturedArgs [][]string
```

Comment:
Not sure I completely understand why this change was necessary.  Did these tests not work before?  How did they change?

Reply:
I was originally changing them to use with Preview, Up, etc, because there isn't an easy way to capture those args as they call runPulumiCmdSync multiple times, so hard to capture and verify those args, but to your point they are no longer relevant for this PR. I'll revert the changes.